src/cli/test/test_cli.c

```diff
@@ -2148,24 +2155,33 @@ static void test_trn_cli_update_transit_network_policy_subcmd(void **state)
 		.bit_val = 10
 	};
 
+	struct rpc_trn_vsip_cidr_t exp_policy2 = {
+		.interface = itf,
+		.tunid = 1,
+		.local_ip = 0x100000a,
+		.cidr_prefixlen = 17,
+		.cidr_ip = 0x60000ac,
+		.cidr_type = 2,
+		.bit_val = 1
+	};
+
+	struct rpc_trn_vsip_cidr_t policies[2];
```

Comment:
nit: could have initialized elements on definition of array

Reply:
will fix